flink-table/flink-sql-gateway/src/test/resources/sql/table.q

```diff
@@ -31,12 +31,12 @@ org.apache.flink.table.api.ValidationException: Table with identifier 'default_c
 
 describe non_exist;
 !output
-org.apache.flink.table.api.ValidationException: Tables or views with the identifier 'default_catalog.default_database.non_exist' doesn't exist
+org.apache.flink.table.api.ValidationException: Tables or views with the identifier 'default_catalog.default_database.non_exist' doesn't exist.
 !error
 
 desc non_exist;
 !output
-org.apache.flink.table.api.ValidationException: Tables or views with the identifier 'default_catalog.default_database.non_exist' doesn't exist
+org.apache.flink.table.api.ValidationException: Tables or views with the identifier 'default_catalog.default_database.non_exist' doesn't exist.
```

Comment:
it looks like same should be applied for sql client's version

Reply:
+1 flink-sql-client-test need to add this either. then CliClientITCase will call to run it.

Reply:
Yes, this is missed. I just fixed it and rebased it to master.